service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerTest.java

```diff
@@ -115,7 +115,21 @@ void shouldBuildNotificationTemplatesWithCompleteValues() throws Exception {
             .put("orders2", "")
             .put("orders3", "")
             .put("orders4", "")
-            .put("directionsAndInterim", "^Information on the whereabouts of the child")
+            .put("orders5", "")
```

Comment:
Please revert addition of extra orders from live templates. Instead of amending existing templates please make template copy and link it from an application.

Adding placeholders to existing templates without sending all data with the request will break live applications (local env on other machines, AAT, DEMO and PROD).